stdlib/std.jsonnet

```diff
@@ -535,7 +535,7 @@ limitations under the License.
       local whole = std.floor(n_);
       local dot_size = if prec == 0 && !ensure_pt then 0 else 1;
       local zp = zero_pad - prec - dot_size;
-      local str = render_int(std.sign(n__) * whole, zp, 0, blank, sign, 10, '');
+      local str = render_int(n__, zp, 0, blank, sign, 10, '');
```

Comment:
Could you modify the comment for render_int explaining what it does with non-whole numbers? This behavior is counter-intuitive, so it's better to have it spelled out explicitly.

Reply:
Have attempted to do it less hackily now